packages/perseus/src/perseus-types.ts

```diff
@@ -664,6 +664,17 @@ export type LockedPointType = {
     filled: boolean;
 };
 
+export type LockedLineType = {
+    type: "line";
+    kind: "line" | "ray" | "segment";
+    points: [startPoint: LockedPointType, endPoint: LockedPointType];
+    color: string;
```

Comment:
Can we make this the type union of available colors instead?

Reply:
Doing this in the LockedPoint PR so it can apply to both 👍🏽